routes/api/user/update-user.js

```diff
@@ -7,5 +7,50 @@ var router = express.Router();
 
 console.log('update-user');
 
+router.get("/api/users/:id", function(req, res) {
+    // Add sequelize code to find a single post where the id is equal to req.params.id,
+    db.Users.update({
+      user_name: req.body.user_name,
+      email: req.body.email,
+      about_me: req.body.about_me,
+      homepage_url: req.body.homepage_url,
+      linkedin_url: req.body.linkedin_url,
+      twitter_url: req.body.twitter_url,
+    }, {
+      where: {
+        id: req.body.id
+      }
+    })
+      .then(function(dbUser) {
+        res.json(dbUser);
+      });
+    // return the result to the user with res.json
+  });
+  router.post("/api/users/password", function(req, res) {
+    db.Users.update({
+      password: req.body.password
+    }, {
+      where: {
+        id: req.body.id
+      }
+    })
+      .then(function(dbUser) {
+        res.json(dbUser);
+      });
+  });
+  router.post("/api/users/password", function(req, res) {
```

Comment:
`router.post` should be `router.put`

`/api/users/password` should be `/api/users/rating`